src/python_testing/matter_testing_support.py

```diff
@@ -117,6 +121,76 @@ def get_default_paa_trust_store(root_path: pathlib.Path) -> pathlib.Path:
         return pathlib.Path.cwd()
 
 
+def parse_pics(lines=typing.List[str]) -> dict[str, bool]:
+    pics = {}
+    for raw in lines:
+        line, _, _ = raw.partition("#")
+        line = line.strip()
+
+        if line == "":
```

Comment:
nit: I believe `if not line` is more pythonic.

Reply:
changed. Also added tests for blank lines and comments which I just realized were missing.